RPC offchain-nonce key

CAGS295 opened the pull request:
Description of proposed changes:

Get an account's offchain nonce key from the node.

Practical tips for PR review & merge:

- [ ] All GitHub Actions report PASS
- [ ] Newly added code/functions have unit tests
  - [ ] Coverage tools report all newly added lines as covered
  - [ ] The positive scenario is exercised
  - [ ] Negative scenarios are exercised, e.g. assert on all possible errors
  - [ ] Assert on events triggered if applicable
  - [ ] Assert on changes made to storage if applicable
- [ ] Modified behavior/functions - try to make sure above test items are covered
- [ ] Integration tests are added if applicable/needed

CAGS295 committed:
RPC offchain nonce key

CAGS295 committed:
bump version

atodorov commented on node/rpc/src/task.rs:
```diff
@@ -0,0 +1,61 @@
+			message: "Unable to query offchain nonce key.".into(),
+			data: Some(format!("{:?}", e).into()),
+		})
+	}
```
No test coverage here at all. Maybe try exercising the error handling as well as a simple positive scenario? 

Or let me know if you'd prefer that I push other commits to this PR instead.

CAGS295 replied:
Feel free to push to my branches anytime, just ping me so that I don't trample your work. 

Regarding coverage, let's discuss how to test the rpc calls.

CAGS295 replied:
WIP

atodorov committed:
Add integration test for getOffchainNonKey() RPC method

Notes:
- use WebSocket and send the JSON-RPC directly b/c rpc.api fails when
  the node returns an error and I haven't found a good way to catch it

0xjeremyfrank commented on node/rpc/src/task.rs:
```diff
@@ -0,0 +1,61 @@
+	C::Api: TaskApi<B, AccountId>,
+	B: traits::Block,
+{
+	fn offchain_nonce_key(&self, acc: String) -> RpcResult<Vec<u8>> {
```
`acc` should be more descriptive. `accountId`

atodorov committed:
ci: Fix a misplaced continue statement

According to Carlos:
> For this case, the CI should pass. No extrinsic changes and there is an impl bump

https://gluwa.slack.com/archives/C03MQ532BGA/p1661451774797509?thread_ts=1661448168.492669&cid=C03MQ532BGA
